src/pages/tea.js

```diff
@@ -0,0 +1,26 @@
+import {products} from '../../backend/db';
+import FilterBar from '../components/FilterBar';
+import Header from '../components/Header';
+import ProductCard from '../components/ProductCard';
+import StyledProductList from '../components/ProductList/styled';
+
+export default function showTea() {
```

Comment:
Components should start with a capitalize letter.

```suggestion
export default function ShowTea() {
```